### ISSUE TITLE
TO BE ABLE TO SHRINK GAMES WITHOUT COPY TO SD

### ISSUE THREAD

**Issue by bman83** [Louhike/Ugly-GDEmu-Games-Manager#35]
Hi Your program is great. 

Please allow game shrink option without copy to sdcard

**Comment by Louhike** [comment]
So, you would like it to shrink games on your PC directly, that's it?

**Comment by bman83** [comment]
Hi Lou,

Thanks for emailing me

Yes,

I managed to use a python script. but I had to install python and git bash
see this youtube video. https://www.youtube.com/watch?v=C9A9ngXgAJo&t=1s

Having a all in one process will be ideal


have you had a look at madsheep sd card gdemu manager?

It also lets you drag and drop game files, rename, and sort the menu

Thanks for your effort writing this software it's great work thankyou


cheers Bman

**Comment by Louhike** [comment]
Yes, I have looked at it and I'm planning on adding all those features in future versions. I'll will also work on adding the ability to shrink games on the PC. I'll tell you when I know when I'll add them in the tool.
